src/components/navigation/GlobalNavigation/GlobalNavigation.stories.tsx

```diff
@@ -98,6 +98,9 @@ const defaultManagement: IGlobalNavigationItem[] = [
       { label: 'option 1', hrefOptions: { href: '/' } },
       { label: 'option 2', hrefOptions: { href: '/' } },
       { label: 'option 3', hrefOptions: { href: '/' } },
+      { label:"button", type: 'button', buttonOptions: { label:"button", onClick:() => alert("go")} },
+      { label:"go", type: 'button', buttonOptions: { label:"go", href:"/", target:"_blank" }}  
```

Comment:
Looks like prettier is not running cause it should take care of that.

```suggestion
      { label:"button", type: 'button', buttonOptions: { label: "button", onClick: () => alert("go")} },
      { label:"go", type: 'button', buttonOptions: { label: "go", href: "/", target: "_blank" }}  
```